detox/src/devices/Device.js

```diff
@@ -103,6 +105,10 @@ class Device {
     }
   }
 
+  setLaunchArg(key, value) {
+    this._launchArgs[key] = value;
```

Comment:
Can we validate against non-strings?
1. if string -> set
2. if `false | undefined | null` -> unset
3. if something else -> throw
?

So it fits with the rest of the logic....

Reply:
TBH, I couldn't entirely make up my mind about this. Need to reiterate, because:
1. We accept all primitives (booleans, numbers apply as well), and also objects and arrays ([ref1](https://github.com/wix/Detox/blob/37a7e038d3510dee3dbc89061e76d02357a01751/detox/test/e2e/22.launch-args.test.js#L15) [ref2](https://github.com/wix/Detox/blob/37a7e038d3510dee3dbc89061e76d02357a01751/detox/test/e2e/22.launch-args.test.js#L30))
2. I agree regarding undefined & null. However I'm generally in favour of making things explicit, so - would it better to throw in a `clearLaunchArg()`?...